extensions/kafka/src/main/java/io/deephaven/kafka/ingest/FieldCopierAdapter.java

```diff
@@ -3,117 +3,25 @@
  */
 package io.deephaven.kafka.ingest;
 
+import io.deephaven.processor.functions.ObjectProcessorFunctions;
 import io.deephaven.qst.type.BoxedBooleanType;
-import io.deephaven.functions.ToBooleanFunction;
-import io.deephaven.functions.ToByteFunction;
-import io.deephaven.functions.ToCharFunction;
-import io.deephaven.functions.ToDoubleFunction;
-import io.deephaven.functions.ToFloatFunction;
-import io.deephaven.functions.ToIntFunction;
-import io.deephaven.functions.ToLongFunction;
-import io.deephaven.functions.ToObjectFunction;
-import io.deephaven.functions.ToPrimitiveFunction;
-import io.deephaven.functions.ToShortFunction;
-import io.deephaven.functions.TypedFunction;
-
-public enum FieldCopierAdapter
-        implements TypedFunction.Visitor<Object, FieldCopier>, ToPrimitiveFunction.Visitor<Object, FieldCopier> {
-    INSTANCE;
+import io.deephaven.function.ToBooleanFunction;
```

Comment:
I bet a lot of these are unused now. Do we even need the class?

Reply:
Removed these unused imports. FieldCopierAdapter is a "bridge" between the world of TypedFunction and the existing kafka implementation; I have hopes we can deprecate existing kafka impl soon and move to one that is ObjectProcessor-first.